scripts/poe/lib/utils.js

```diff
@@ -22,10 +39,17 @@ module.exports = {
   mmVersion: () => {
     return pkg.version;
   },
-  log: (msg) => {
-    console.log(msg);
-  },
-  error: (msg) => {
-    console.log(`${chalk.red('Error: ')}${msg}`);
+  save: async (fileUrl, filePath) => {
+    const res = await get({url: fileUrl, encoding: 'utf-8'});
+    /*
+      - Sort file content alpahabetically in ascending order
+      - Get rid of comments (lines starting with #)
+    */
+    const out = res.body
+      .split(os.EOL)
+      .filter(line => !line.startsWith('#'))
+      .sort()
+      .join(os.EOL);
```

Comment:
I think we should stick to only one kind of line ending, otherwise if it's run on Windows every line in the file will be updated and the diff will be useless. I think just hardcode the `split` and `join` lines to `\n`.